hack/csv-generate.sh

```diff
@@ -83,8 +83,8 @@ if [[ ${CSV_VERSION} =~ 4.7.* ]]; then
     --replaces-csv-version "${REPLACES_CSV_VERSION}" \
     --skip-range "${CSV_SKIP_RANGE}" \
     --min-kube-version "${MIN_KUBE_VERSION}" \
-    "${MAINTAINERS}" \
-    "${EXTRA_ANNOTATIONS}"
+    "${EXTRA_ANNOTATIONS}" \
```

Comment:
You changed the order, in purpose?

Reply:
yes, and also renamed. The order is not really relevant here. But I can split to a different commit if needed.

Reply:
No need, thanks